### ISSUE TITLE
Skipping methods containing rvalue reference parameters

### ISSUE THREAD

**Issue by reubenjohn** [CastXML/pygccxml#153]
Notice the rvalue reference parameter **B&&** of the **MyClass::update** method shown below. It causes the update method to be skipped in the castxml XML output entirely. Simply making it a lvalue reference **B&** causes it to pop up in the output.

```
namespace test {
class A { };
class B { };

template<typename T1>
class MyClass {
...
void update(B&& b) { ... }
};

// Explicit instantiation
template void MyClass<A>::update(B&&);
}
```
The templatization may be unnecessary to reproduce this issue.

**Comment by reubenjohn** [comment]
If it isn't being generated in the XML file, it is likely the case that this is a bug originating from CastXML, so I've [duplicated](https://github.com/CastXML/CastXML/issues/242) this issue there. Happy to close this one if you agree.

**Comment by reubenjohn** [comment]
The issue originated in castxml and the fix is now present on the master branch: https://github.com/CastXML/CastXML/issues/242#issuecomment-1442458423
I'm waiting to hear on when it will be released so that it can be used here. If I notice the release, I will update this issue.

**Comment by iMichka** [comment]
This is fixed in release 0.6 of CastXML: https://github.com/CastXML/CastXML/pull/243
